src/main/java/co/rsk/federate/signing/hsm/HSMVersion.java

```diff
@@ -22,14 +21,14 @@ public int getNumber() {
     }
 
     public static List<HSMVersion> getPowHSMVersions() {
-        return Arrays.asList(V2, V3, V4, V5);
+        return Arrays.asList(V2, V4, V5);
     }
 
     public static boolean isPowHSM(int version) {
         return getPowHSMVersions().stream().anyMatch(hsmVersion -> hsmVersion.number == version);
     }
 
-    public static boolean isPowHSM(HSMVersion version){
-        return getPowHSMVersions().contains(version);
+    public static boolean isValidHSMVersion(int version) {
+        return Arrays.stream(values()).anyMatch(hsmVersion -> hsmVersion.number == version);
     }
```

Comment:
```suggestion
    public static boolean isValidHSMVersion(HSMVersion version) {
        return Arrays.stream(values()).contains(version);
    }
```

wdyt? I think the method should receive an HSMVersion, not any random int